package.json

```diff
@@ -116,9 +117,8 @@
     }
   },
   "detox": {
-    "test-runner": "mocha",
+    "test-runner": "jest",
     "specs": "e2e",
-    "runner-config": "e2e/mocha.opts",
```

Comment:
Delete `e2e/mocha.opts`?

Reply:
Can we remove mocha from devDependencies now?

Reply:
I'll remove it.

Reply:
And with jest we use e2e/config.json